forc/src/cli/commands/build.rs

```diff
@@ -48,6 +48,15 @@ pub struct Command {
     /// needs to be updated, Forc will exit with an error
     #[clap(long)]
     pub locked: bool,
+    /// Name of the build profile to use.
+    /// If it is not specified, forc will use debug build profile.
+    #[clap(long)]
+    pub build_profile: Option<String>,
```

Comment:
I think we can name this option `profile`, as the `build` prefix is already implied by the command. E.g.

```
forc build --profile <profile>
```
rather than
```
forc build --build-profile <profile>
```